src/doc/en/developer/packaging.rst

```diff
@@ -1366,3 +1207,220 @@ must meet the following requirements:
   :ref:`chapter-github`.
 
 
+.. _section-spkg-patching:
+
+Modifying third-party code
+==========================
+
```

Comment:
I think "Inclusion procedure for new and updated packages" section should be behind this section.

Reply:
I don't know, I put this section last in this chapter because the intention is that this is a rare event

Reply:
But that section is still about preparing the spkg, which happens before "Inclusion procedure for new and updated packages".

Reply:
If that was a "footnote", then it would go to the bottom. But it is not the case.

Reply:
OK, I've moved it